shared/constants/types/fs.js

```diff
@@ -6,3 +6,21 @@ export type _State = {
 }
 
 export type State = I.RecordOf<_State>
+
+export type FolderVisibility = 'private' | 'public' | 'team' | null
+
+export type FolderHeaderProps = {
```

Comment:
so typically constants/types contains things related to the store only. So i imagine we will have FolderVisibiilty as metadata on a path, but FileRowProps is just props for a single row and likely belongs with the dumb component (fs/rows/file.js or whatever)